### DISCUSSION TITLE
Regarding Download Instructions for ISO files

### DISCUSSION THREAD

**Discussion by nagu165** [agama-project/agama#968] [General]
Hello sir, In the ReadMe page of this Repo the link provided to the download page of ISO files doesn't provide an idea of which file is to be downloaded according to the requirements of the user. It took me a while to figure out the right download file. Hence I think it would be better to update the ReadMe file or the download page to provide additional information.
![Screenshot 2023-12-30 191442](https://github.com/openSUSE/agama/assets/125825782/1c598300-c6b7-46bf-b2c2-fa9d8956c7b7)
![Screenshot 2023-12-30 192238](https://github.com/openSUSE/agama/assets/125825782/4ecad666-22e7-408d-996d-8c9c4f24176b)
I would like to work on the issue with your help if permitted.
Thank you.

**Comment by dgdavid** [comment]
I had overlooked this until now. Of course, any kind of contribution for improving the project is welcome, READMEs included.
